### ISSUE TITLE
Implement Big Picture / TV Mode with Controller Support

### ISSUE THREAD

**Issue by firelad97** [FaithLV/Spectabis#37]
Hello!

I would love to see a Big Picture implemented in this program as I use my TV often when I play games. Similar to PCSX2Bonus's TV Mode and Steam's Big Picture, I could navigate around in the menu with the controller and select the game whatever I want to play.

I also would like to see some UI menus when configuring the games in Big Picture, rather than using the default PCSX2 menu. Some menu functions aren't needed, but you know what I mean. Configuring global controller setting, video plugin setting, and such are the most important.

Looking forward to this! Let me know if this is possible or not. Thanks!

**Comment by FaithLV** [comment]
I would also love to see the Big Picture implemented, but as you can see, I'm not really working on this anymore. 

I don't have that much free time and I haven't played any PS2 game in a while and even if I did, I'm not using Windows anymore. 

But I'll leave this open, because somebody might work on it someday.

**Comment by Michanne** [comment]
It is possible, though currently there is no controller support built-in for the UI, regardless of Big Picture or not. This will be some time to implement as very few people are working on this project..

**Comment by FaithLV** [comment]
Yeah, there's no easy way to implement it. I had started some work, but found none elegant ways of doing so. I think XInput listener is fully working and still there, but that's the least of the issues of getting this to work.
